lab_1/main.py

```diff
@@ -69,9 +128,34 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance = []
+    left_context = []
+    right_context = []
+    if type(tokens) == list and type(word) == str and type(left_context_size) == int and type(right_context_size) == int:
```

Comment:
Usually, there are check for input arguments and body function - on the same level.
No need for nesting.